moped-editor/src/components/GridTable/GridTableSearch.js

```diff
@@ -123,53 +123,13 @@ const GridTableSearch = ({
     query.config.useQuery
   );
 
-  /**
-   * Generates a sanitized string for CSV
-   * @param {*} value - Any value
-   * @return {string}
-   */
-  const dataSanitizeValueExport = (value) => {
-    return typeof value !== "number" ? `"${value}"` : String(value);
-  };
-
-  /**
-   * Retrieves a list of headers for the data
-   * @param {Array} data - The data payload
-   * @return {string[]}
-   */
-  const dataGetHeaders = (data) => {
-    return Array.isArray(data)
-      ? Object.keys(data[0]).filter((key) => key !== "__typename")
-      : [];
-  };
-
-  /**
-   * Converts a single data entry into a CSV line
-   * @param {string[]} headers - The list of headers that determines the order
-   * @param {Object} data - The data as provided by Hasura
-   * @return {string}
-   */
-  const dataToCSV = (headers, data) => {
-    return (
-      headers.join(",") +
-      "\n" +
-      data
-        .map((item) => {
-          return headers
-            .map((key) => dataSanitizeValueExport(item[key]))
-            .join(",");
-        })
-        .join("\n")
-    );
-  };
-
   /**
    * Downloads the contents of fileContents into a file
    * @param {string} fileContents
    */
   const downloadFile = (fileContents) => {
     const exportFileName =
-      query.table + format(Date.now(), "yyyy-MM-dd'T'HH:mm:ssxxx");
+      "moped-" + query.table + format(Date.now(), "yyyy-MM-dd'T'HH:mm:ssxxx");
```

Comment:
what do yall think of adding moped to the filename?

Reply:
I like it!  I could see a lot of users juggling exports from different apps and knowing where this one comes from could be helpful.

Reply:
agreed!